src/cowboy_http.erl

```diff
@@ -246,7 +246,7 @@ loop(State=#state{parent=Parent, socket=Socket, transport=Transport, opts=Opts,
 			loop(State);
 		%% System messages.
 		{'EXIT', Parent, Reason} ->
-			terminate(State, {stop, {exit, Reason}, 'Parent process terminated.'});
+			initiate_closing(State, Reason);
```

Comment:
Please restore the stop tuple so the exit reason is the same for HTTP/1.1 and HTTP/2.

Reply:
OK, will do.

Btw, for HTTP/2 I added a separate clause for `shutdown` but not here. Should I? I guess we don't want graceful shutdown if the parent exits with other reasons, e.g. `killed`.

Reply:
I'm OK with no graceful shutdown for other reasons for the time being.